src/pages/catalog-page/CatalogPage.tsx

```diff
@@ -35,33 +32,26 @@ const paginationStyles = {
 }
 
 const CatalogPage = () => {
-  const { params, page } = useParams()
-  const [offset, setOffset] = useState<number>(parseInt(page!))
-  const [operationType, setOperationType] =
-    useState<FetchItemOperationType>('fetch')
+  const { page }: any = useParams()
+  const [pageN, setPageN] = useState(parseInt(page) || 1)
```

Comment:
why is it called `pageN` and not just `page`?

Reply:
Initially, pageN was used solely for page numbering, page !== pageN, page was used for pagination.
Now there is no need to pass the page number in the URL because the meeting said that pages in the catalog should not be accessible by link, so I fixed it